README.md

```diff
@@ -37,16 +37,17 @@ Once you're done, learn how to set up your local environment with the
 [quickstart guide](./examples/gno.land/r/demo/boards/README.md) and the
 [contributing guide](./CONTRIBUTING.md).
 
-You can find out more existing tools & documentation for Gno on our
-[awesome-gno](https://github.com/gnolang/awesome-gno) repository.
-We look forward to seeing your first PR!
+You can discover additional details about current tools and Gno documentation on
+our official documentation page. Additionally, the [awesome-gno](https://github.com/gnolang/awesome-gno)
```

Comment:
Why not add a link to the docs here as well, like it's added for awesome gno?

Reply:
[1a518f3](https://github.com/gnolang/gno/pull/1520/commits/1a518f3d97dcfe78021c67df8525a64cfe00f588)